mcp/infra/__main__.py

```diff
@@ -138,11 +147,17 @@ def get_ssm_parameter_arn(parameter_name):
 )
 
 aws.iam.RolePolicyAttachment(
-    f"{application_name}-ecs-task-execution-role-policy-attachment",
+    f"{application_name}-ecs-exec-policy",
     role=ecs_task_execution_role.name,
     policy_arn="arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy",
 )
 
+# Create a CloudWatch log group for the application logs
+log_group = aws.cloudwatch.LogGroup(
+    f"{application_name}-logs",
+    retention_in_days=7,
+)
+
```

Comment:
interesting about logs – if this does get used by the policy team, I imagine good logs could be really valuable to us (ie worth not deleting them after a week)? I wonder whether it's worth doing a little observability afternoon-sprint on this (whether cloudfront or not)